pyomo/pysp/tests/convert/test_ddsip.py

```diff
@@ -85,15 +74,14 @@ def _get_cmd(self,
     @unittest.nottest
     def _run_bad_conversion_test(self, *args, **kwds):
         cmd, output_dir = self._get_cmd(*args, **kwds)
-        outfile = output_dir+".out"
-        rc = pyutilib.subprocess.run(cmd, outfile=outfile)
-        self.assertNotEqual(rc[0], 0)
-        self._assert_contains(
-            outfile,
-            b"ValueError: One or more deterministic parts of the problem found in file")
+        result = subprocess.run(cmd, stdout=subprocess.PIPE,
+                                stderr=subprocess.PIPE)
+        self.assertNotEqual(result.returncode, 0)
+        self.assertIn(
+           b"ValueError: One or more deterministic parts of the problem found in file",
+           result.stderr)
```

Comment:
Out of curiosity, is there a reason to look for a binary string instead of decoding stderr?

Reply:
No particular reason (besides that it was less typing). I'm not married to this choice if you want me to decode stderr instead.